src/qualisys-to-ros.cpp

```diff
@@ -0,0 +1,161 @@
+/**
+ *
+ */
+
+#include "ros-qualisys/qualisys-to-ros.hpp"
+
+#include <cmath>
+
+namespace ros_qualisys {
+
+QualisysToRos::QualisysToRos() {}
+
+QualisysToRos::~QualisysToRos() {}
+
+void QualisysToRos::setNodeHandle(ros::NodeHandle& nh) {
+  node_handle_ = std::addressof(nh);
+}
+
+bool QualisysToRos::initialize() {
+  assert(node_handle_ && "Wrong initialization of the node handle.");
+  // The base port (as entered in QTM, TCP/IP port number, in the RT output tab
+  // of the workspace options
+
+  int tmp_base_port(0);
+  int tmp_udp_port(0);
+  node_handle_->param("server_address", server_address_,
+                      std::string("192.168.0.1"));
```

Comment:
```suggestion
                      std::string("localhost"));
```

Or `127.0.0.1` if you need a numeric IPv4, but 192.168.0.1 is most of the time not a sensible default value